sherlock/notify.py

```diff
@@ -185,6 +184,20 @@ def finish(self, message="The processing has been finished."):
 
         # return
 
+    def countResults(self):
+        """This function counts the number of results. Every time the fuction is called,
+        the number of results is increasing.
+
+        Keyword Arguments:
+        self                   -- This object.
+
+        Return Value:
+        The number of results by the time we call the function.
+        """
+        global globvar
+        globvar += 1
+        return globvar
+        
```

Comment:
for every social media in which the username exists, the "globvar" parameter is increasing.